rush.json

```diff
@@ -789,6 +789,11 @@
       "packageName": "@azure-tests/perf-app-configuration",
       "projectFolder": "sdk/appconfiguration/perf-tests/app-configuration",
       "versionPolicyName": "test"
+    },
+    {
+      "packageName": "@azure/mixedreality-remoterendering",
```

Comment:
The MixedReality naming groups a related set of services. When working on the .net SDK, I originally had them in the same folder, but I was instructed [to move them out](https://github.com/Azure/azure-sdk-for-net/pull/18905). IIRC, the infrastructure imposes constraints on the timing of how sibling SDKs are released, which turned out to be a problem.

Reply:
Well the only hassle is the release pipeline is shared, so for JS at least you have to deal with cancelling the release of the packages you aren't publishing (we do this all the time for core where we only want to update 1 package.)

For our sanity, I don't want to play this game of "find the package" where the `sdk/<group>/` has nothing to do with the package name itself

Reply:
I wasn't party to the full discussion about moving remoterendering to its own folder (when that decision was made for .net). @craigktreasure, can you remember the details and who was involved?

@xirzec: Can I confirm that you are happy if JS is inconsistent with the other SDKs in this respect?

Reply:
@MalcolmTyrrell I'm leaning towards yes, but now that @bterlson is back, I'd be interested in his opinion.

Good point about form recognizer and text anlaytics, though in those cases we don't have a `sdk/ai` folder to add confusion.

Reply:
To be consistent, we at least need to use vanilla kebab-case for this package name. NPM doesn't have namespaces, and the convention that uses dash as a namespace separator instead of a space is not common and often confusing. In the last couple years we've been fairly consistent about not using it.

I do find the mixed-reality prefix to be useful however (remote rendering being a fairly generic technology, with this library offering something more specific). Would `mixed-reality-remote-rendering` be acceptable?

Reply:
Done! I have renamed the folder and package mixed-reality-remote-rendering. For consistency, I performed the same operation on mixedreality-authentication, which is now called mixed-reality-authentication (we haven't released packages for the mixed-reality-authentication sdk yet, so no one should be affected).

Reply:
Thank you for doing this! I find the new name much easier to read